src/Modules/Data/Commands.Data/GetPowerBIDataset.cs

```diff
@@ -53,6 +53,15 @@ public class GetPowerBIDataset : PowerBIClientCmdlet, IUserScope, IUserFilter, I
         [Parameter(Mandatory = false, ParameterSetName = ObjectListParameterSetName)]
         public int? First { get; set; }
 
+        [Alias("Expand")]
+        [Parameter(Mandatory = false, ParameterSetName = IdParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = NameParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = ListParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = ObjectIdParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = ObjectNameParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = ObjectListParameterSetName)]
+        public string Include { get; set; }
```

Comment:
Either add a ValidateSet or do an array of enums here

Reply:
we'll start with actualStorage and add other later